src/index.md

```diff
@@ -4,91 +4,76 @@ Welcome to the Hyperledger Iroha 2 tutorial. This document is designed to help y
+The first version of Iroha used a consensus algorithm called _Yac_. It is _crash-fault-tolerant_, which means that it can survive a set number of nodes _crashing_: i.e. losing power, being cut off from the network, or being destroyed with a sledgehammer. _Sumeragi_, by contrast, was designed to be _Byzantine-fault-tolerant_. This means that Iroha 2 can tolerate not only peers being inactive on the network, but also running malicious software, and actively trying to falsify data in the blockchain.
 
-Of course, if this is not sufficiently low-level for your particular needs, you can fork Iroha, since it's licensed under the Apache 2.0 permissive license and is part of the Hyperledger Foundation, which itself is a subsidiary of the Linux foundation.
+We can mathematically prove that **Iroha 2 can work when up to 33% of its nodes are actively trying to stop Iroha 2 from working properly** (_or at all_). In other words, even if someone gained control of a _third_ of all of your network nodes, an Iroha 2 deployment is _mathematically guaranteed_ to keep working.
+:::
 
-Iroha 2 is an event-driven blockchain. Each change in the state of the blockchain is accompanied by its own event that can trigger a smartcontract: complex logic that allows for on-chain scripting. For smartcontracts, Iroha 2 supports two approaches.
+Iroha 2 is a minimalist code base. We take great care to vet our dependencies, and avoid large inter-dependent chunks of code. We provide a few telemetry APIs, including `prometheus` tooling, structured logging in JSON, as well as compatibility with [standard tools](https://wiki.sora.org/sora-faq) used in Parity Substrate. Our data is strongly-typed, and our methods — statically dispatched. We make use of the best that Rust has to offer: `serde` and `parity_scale_codec` for serialisation, `tokio` for co-operative multi-threading, as well as wealth of testing, bench-marking, static analysis and code auditing tools that come packaged with the exemplary `cargo`. Our code is easy to reason about, and quick to compile, whilst also being ergonomic to use and thoughtfully crafted. We have no `panics` and no `unsafe` code.
 
-- Iroha Special Instructions
-- WASM
+Iroha 2 is also more flexible than the original Iroha due to modular design. It is possible to add or remove features based on the particular use-case.  If you want to be extremely fast, and work on embedded hardware, just compile Iroha 2 without the `expensive-metrics` feature. Don't use telemetry at all? Remove it entirely and enjoy even more performance. _Permission_ sets are plugins that can be upgraded during run-time. We have an extensive module system as well as a robust WASM runtime framework.
```

Comment:
For the record, two spaces after a full stop is preferred in plain text formats such as LaTeX, Org and RST, but if I'm the only one following this convention it will lead to confusion for everyone who doesn't.